R/create_individual_file.R

```diff
@@ -483,8 +483,10 @@ add_ch_columns <- function(episode_file, prefix, condition) {
       ch_ep_end = dplyr::if_else(
         eval(condition),
         .data$record_keydate2,
-        lubridate::NA_Date_  ),
-      # If end date is missing use the first day of next FY quarter
+        lubridate::NA_Date_
+      ),
+      # check logic here for care home methodology
+      # If end date is missing use the end of the FY quarter
```

Comment:
Check care home logic here and decide if we need the quarter calculation functions